src/test/java/org/broadinstitute/consent/http/service/dao/SamDAOTest.java

```diff
@@ -39,9 +39,9 @@
 import static org.mockserver.model.HttpRequest.request;
 import static org.mockserver.model.HttpResponse.response;
 
-public class SamServiceTest implements WithMockServer {
+public class SamDAOTest implements WithMockServer {
 
-  private SamService service;
+  private SamDAO service;
```

Comment:
since it's blocking quazi, we can merge ti as is, but nit: rename this to `samDao`? perhaps quazi you could do this in your ticket.

Reply:
Just getting to this, sorry. Merging after tests pass 👍🏽